Podfile

```diff
@@ -20,7 +20,7 @@ target 'ios-showcase-template' do
   pod 'SwiftKeychainWrapper'
   pod 'NotificationBannerSwift'
   pod 'RealmSwift'
-  pod 'SCLAlertView', :git => 'https://github.com/vikmeup/SCLAlertView-Swift.git', :branch => 'master'
+  pod 'SCLAlertView', :git => 'https://github.com/vikmeup/SCLAlertView-Swift.git', :branch => 'swift-2.3'
```

Comment:
@wei-lee @wtrocki We were unable to get the app to build via the Mobile CI/CD service without changing this. Do you know if we need to use master here or can we use this branch? Are there updates we need to make to the Mobile CI/CD service to allow the master branch to work? It is currently throwing this error:

```
bash-3.2$ pod install                                               
Analyzing dependencies                                              
Pre-downloading: `SCLAlertView` from `https://github.com/vikmeup/SCLAlertView-Swift.git`, branch `master`                               
[!] Unable to find a specification for 'SCLAlertView'.              

[!] Unable to load a podspec from `SCLAlertView.podspec`, skipping: 

Pod::DSLError
```

Reply:
I tried to run `pod install` from my mac and got the same error